tests/integrationv2/utils.py

```diff
@@ -47,15 +48,33 @@ def invalid_test_parameters(*args, **kwargs):
     cipher = kwargs.get('cipher')
     curve = kwargs.get('curve')
 
+    if protocol is Protocols.TLS13 and get_flag(S2N_OPENSSL_VERSION) == "openssl-1.0.2-fips":
```

Comment:
Why is this combination not allowed? Can you add a comment?

Reply:
As mentioned above, it isn't supported.

Reply:
Spoke offline, this check is unnecessary at this time.